### PR TITLE
Re-write Overview page

### PR DESCRIPTION
**Pull request by shaniR**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/getting-started/basic-architecture.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### Components
+ 
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
+
+ The Key Manager is the identity provider for the WSO2 API Manager and acts as the Secure Token Service(STS). 
+ 
+ In WSO2 API Manager, tokens are generated for an application. The Key manager provides a token API to generate access tokens. These tokens can be used by clients to invoke APIs exposed by WSO2 API Manager. Furthermore, 
+ The Key Manager exposes a revoke API to revoke any access token. The Key Manager supports OAuth2.0, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms.
```
Comment (chamilaadhi) [inline en/docs/getting-started/basic-architecture.md:60]: `The Key Manager supports` should be `The API Manager supports`

File: en/docs/getting-started/basic-architecture.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
+
+ The Key Manager is the identity provider for the WSO2 API Manager and acts as the Secure Token Service(STS). 
+ 
+ In WSO2 API Manager, tokens are generated for an application. The Key manager provides a token API to generate access tokens. These tokens can be used by clients to invoke APIs exposed by WSO2 API Manager. Furthermore, 
+ The Key Manager exposes a revoke API to revoke any access token. The Key Manager supports OAuth2.0, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms. 
+ 
+ WSO2 API Manager only supports creating applications that support JWT formatted OAuth2.0 type tokens.
```
Comment (nuwand) [inline en/docs/getting-started/basic-architecture.md:62]: Remove the line above. This is the norm for a majority of major IDPs.

File: en/docs/getting-started/basic-architecture.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ A client can generate a JWT formatted OAuth 2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, an API Key can be generated through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token.
+ 
+ When a client invokes an API with a JWT formatted OAuth2.0 access token or an API Key,  the Gateway validates the token by validating its signature and subscription.
```
Comment (nuwand) [inline en/docs/getting-started/basic-architecture.md:65]: Please remove all references of "JWT formatted access tokens". The format of the token doesn't have to be highlighted in this manner because we aren't doing anything special.

File: en/docs/getting-started/basic-architecture.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
```
Comment (chamilaadhi) [inline en/docs/getting-started/basic-architecture.md:66]: we do validate the scopes and generate backend jwt token as well if it is enabled

File: en/docs/getting-started/basic-architecture.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+ 
+ In WSO2 API Manager, tokens are generated for an application. The Key manager provides a token API to generate access tokens. These tokens can be used by clients to invoke APIs exposed by WSO2 API Manager. Furthermore, 
+ The Key Manager exposes a revoke API to revoke any access token. The Key Manager supports OAuth2.0, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms. 
+ 
+ WSO2 API Manager only supports creating applications that support JWT formatted OAuth2.0 type tokens.
+ A client can generate a JWT formatted OAuth 2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, an API Key can be generated through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token.
+ 
+ When a client invokes an API with a JWT formatted OAuth2.0 access token or an API Key,  the Gateway validates the token by validating its signature and subscription.
+ 
+ WSO2 API Manager 3.2.0 can accommodate a third-party Key Manager in place of its Key Manager component and also accommodate multiple Key Managers such as Okta, OAuth0, KeyCloak, etc.
```
Comment (nuwand) [inline en/docs/getting-started/basic-architecture.md:67]: No need to mention the version of the product here since the docs are any tied to a particular version.

File: en/docs/getting-started/basic-architecture.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+ 
+ 
+#### Traffic Manager
+ 
+  ![Traffic Manager]({{base_path}}/assets/img/get_started/traffic-manager-overview.png)
+ 
+The Traffic Manager helps users to regulate API traffic, make APIs and applications available to consumers at different service levels, and secure APIs against security attacks. The Traffic Manager features a dynamic throttling engine to process throttling policies in real-time, including rate-limiting of API requests. 
+
+In addition to the above, the traffic manager also helps to keep the API Gateway's in-memory map, used for key validation, up-to-date. The traffic manager updates the databases based on artifact(API/application) update events received from the API Publisher and API Developer Portal.  The Traffic Manager then goes on to notify the API Gateway and help it update its in-memory map via REST APIs.
+
```
Comment (chamilaadhi) [inline en/docs/getting-started/basic-architecture.md:77]: I think we need to re-word this paragraph. Couple of this to point out
1- traffic manager does not update the database. it receives events from publisher/devportal on changes such as API create, app create etc and put them in to a jms topic. Gateways which are subscribed to this topic get this message to them and it updates it's inmemory maps based on the message.

File: en/docs/getting-started/basic-architecture.md (hunk 5 above)
Comment (nuwand) [inline en/docs/getting-started/basic-architecture.md:67]: Reword this as "In addition to the built in Key Manger being used as the IDP, the API Manager also supports integrating with third party IDPs such as Okta, Auth0, KeyCloak and so on for OAuth2 based security for APIs"

**Review by nuwand — CHANGES_REQUESTED** [review]

File: en/docs/getting-started/basic-architecture.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+ 
+## Control Plane
+ 
+ Control plane is where the API security validation, API key generation and rate limiting decisions are taken.
+ 
+### Components
+ 
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
```
Comment (nuwand) [inline en/docs/getting-started/basic-architecture.md:55]: The main use-case of the Key Manager is for clients to login and get tokens. These tokens are then sent to the gateway when accessing APIs. This diagram of the Key Manager illustrates an internal architecture of the product where the KM is used by the dev portal and publisher but misses to illustrates the KM's primary use case. I suggest to redraw this image to illustrate the primary use case of the KM and to ignore the product internals to keep it simple.

**Review by nuwand — CHANGES_REQUESTED** [review]

File: en/docs/getting-started/basic-architecture.md (hunk 7 above)
Comment (shaniR) [reply en/docs/getting-started/basic-architecture.md:55]: noted. Will do.

File: en/docs/getting-started/basic-architecture.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+### Components
+
+#### API Gateway
+
+![API Gateway]({{base_path}}/assets/img/learn/gateway-overview.png)
+
+WSO2 API Gateway acts as the entry point for an API request made to an API managed by WSO2 API Manager.
+
+JWT token validation is done by the API Gateway. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map. This in-memory map includes API-related information, application-related information, subscription-related information, etc. which is updated each time an artifact (API/application) is updated.
```
Comment (bhathiya) [inline en/docs/getting-started/basic-architecture.md:43]: it validates the signature, issuer, expiry time, and API subscriptions. and we are using the in-memory map to validate the subscriptions only.

File: en/docs/getting-started/basic-architecture.md [hunk 9]
```diff
@@ -1,73 +1,72 @@
-#Basic Architecture and Key Components
+# Basic Architecture and Key Components
 
-![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)
+[![Basic Architecture]({{base_path}}/assets/img/get_started/basic-architecture.png)]({{base_path}}/assets/img/get_started/basic-architecture.png)
```
Comment (Mariangela) [inline en/docs/getting-started/basic-architecture.md:3]: Why are there diagonal lines in the background of this image?
Screenshot 2020-12-09 at 17 22 08

File: en/docs/getting-started/basic-architecture.md [hunk 10]
```diff
@@ -1,73 +1,72 @@
...
 
-The management plane is where API creation and API management takes place.  It consists of portals (publisher, developer portal, admin portal) for users to create and manage APIs, implement rate-limiting policies, monitor and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The management plane includes API Analytics dashboards, displaying various business insights. . 
+The Management Plane is where API creation and API management takes place.  It consists of portals (Publisher, Developer Portal, Admin Portal) for users to create and manage APIs, implement rate limiting policies, monitor, and monetization, etc. It also provides a set of APIs to interact with external tools like API Controller. The Management Plane includes API Analytics dashboards, displaying various business insights.
 
 ### Components
 
 #### API Publisher
 
-![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
+[![API Publisher]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)]({{base_path}}/assets/img/learn/overviewpage-rest-api.jpg)
```
Comment (Mariangela) [inline en/docs/getting-started/basic-architecture.md:13]: This screenshot is outdated. The menu named "Scopes" should be "Local Scopes"

Screenshot 2020-12-09 at 17 29 04

File: en/docs/getting-started/basic-architecture.md [hunk 11]
```diff
@@ -1,73 +1,72 @@
...
-
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to and consume APIs securely and easily.
+[![Developer Portal]({{base_path}}/assets/img/get_started/developer-portal-overview.png)]({{base_path}}/assets/img/get_started/developer-portal-overview.png)
 
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to, and consume APIs securely and easily.
 
 #### API Analytics
 
-![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)
+[![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)]({{base_path}}/assets/img/get_started/analytics.png)
```
Comment (Mariangela) [inline en/docs/getting-started/basic-architecture.md:25]: Screenshot 2020-12-09 at 17 35 11
It will be great if we can get the following updated in this image
Realtime > Real-Time
tcp connection > TCP connection
HTTP connection > HTTP connection

File: en/docs/getting-started/overview.md [hunk 12]
```diff
@@ -1,99 +1,101 @@
 # Overview
 
-WSO2 API Manager is a fully open-source solution for end to end API Management in the cloud, on-prem or in hybrid environments. It comes with an [Apache Software License Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) which makes it **free to use**. It allows API developers to design, publish, and manage the lifecycle of APIs and API product managers to create API products from one or more APIs. It hosts an application developer portal which helps in building and managing a developer community for your APIs. Its cloud-native API gateway is used for securing, routing, controlling and monitoring your API traffic in a scalable manner.
+<div style="width: 100%; overflow: hidden;">
+  
+<div class="rightContentOverview" style="width:40% !important;"> WSO2 API Manager is a fully open-source API management platform. It supports API designing, API publishing,  lifecycle management, application development, API security,  rate limiting, viewing statistics of APIs, as well as connecting APIs, API Products, and endpoints.
```
Comment (Mariangela) [inline en/docs/getting-started/overview.md:5]: We need to add a bit of space between the text and the video.
Screenshot 2020-12-09 at 17 03 35

File: en/docs/getting-started/overview.md [hunk 13]
```diff
@@ -1,99 +1,101 @@
...
--  Developer Portal
--  Manage Developer Community
--  Manage and Scale API Traffic
--  Monitor and Monetize
--  Pluggable, Extensible, and Themeable
--  Easily Deployable in Your Enterprise
--  WSO2 Platform Multi-Tenancy Support
--  Multiple Key Manager Support
+<div class="leftContentOverview" style="width:75% !important;">A well-designed API can make your APIs easy to use. WSO2 API Manager’s API Publisher guides you through API creation to API Publishing while adhering to the respective API’s specification. The API Publisher‘s user-friendly UI helps you create APIs, manage the API life cycle, secure your APIs, apply rate limiting policies, connect to endpoints, and publish your APIs easily. Additionally, you can create and publish API Products as well.
+  </div>
```
Comment (Mariangela) [inline en/docs/getting-started/overview.md:19]: Can we reduce the white space between the image and the text. This applies to all the images.
Screenshot 2020-12-09 at 17 06 50

File: en/docs/getting-started/overview.md [hunk 14]
```diff
@@ -1,99 +1,101 @@
...
+<div style="width: 100%; overflow: hidden;">
 
-The API Gateway supports OAuth 2.0, JWT, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms the above-mentioned authentication mechanisms and enables IT organizations to enforce rate limits and throttling policies.
+<div class="leftContentOverview" style="width:75% !important;">Balancing the load of your system is critical to avoid system outages. WSO2 API Manager provides the capability to add rate limiting policies to your APIs. Furthermore, you can use these policies to monetize your APIs and bring revenue to your organization.  
+  </div>
+  
+  <div class="rightImageOverview">
```
Comment (Mariangela) [inline en/docs/getting-started/overview.md:81]: Can we get marketing to redraw this image as it shows that it does not follow the same standard as the other images. As the other images have thinner lines.
Screenshot 2020-12-09 at 17 46 48

File: en/docs/getting-started/basic-architecture.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+#### API Analytics
+
+[![API Analytics]({{base_path}}/assets/img/get_started/analytics.png)]({{base_path}}/assets/img/get_started/analytics.png)
+
+Monitoring and analytics of APIs deployed on WSO2 API Manager are provided by the WSO2 API Manager Analytics component. This component includes different types of dashboards to provide more detailed different statistical views of your APIs. Furthermore, you can secure your system by configuring alerts and notifications on pre-determined events to identify any unusual behaviors in near real-time.  For more information, see [Working with Analytics]({{base_path}}/learn/analytics/configuring-apim-analytics).
+
+
+## Data Plane
+
+The Data Plane is where the created API is exposed to the public consumers and acts as the proxy for API calls. This also provides additional capabilities such as enforcing security, rate limiting etc. (policy enforcement point).
```
Comment (Mariangela) [inline en/docs/getting-started/basic-architecture.md:32]: Why do we say "(policy enforcement point)" here? It looks like it is part of the draft and needs to be removed.

File: en/docs/getting-started/basic-architecture.md [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+
+### Components
+
+#### API Gateway
+
+[![API Gateway]({{base_path}}/assets/img/learn/gateway-overview.png)]({{base_path}}/assets/img/learn/gateway-overview.png)
+
+WSO2 API Gateway acts as the entry point for an API request made to an API managed by WSO2 API Manager.
+
+The API Gateway does the JWT token validation. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map. This in-memory map includes API-related information, application-related information, subscription-related information, etc. which is updated each time an artifact (API/application) is updated.
```
Comment (Mariangela) [inline en/docs/getting-started/basic-architecture.md:42]: This sentence needs to be revisited as it is incorrect.
"The API Gateway does the JWT token validation. The API Gateway validates it's signature and subscription using the information it has in it's an in-memory map."

**Review by Mariangela — CHANGES_REQUESTED** [review]

**Comment by Mariangela** [comment]
@Shani, please note that I reviewed and updated the documentation with the following commits. I have also added some comments for you to address.
- https://github.com/wso2/docs-apim/pull/2485/commits/8514df9d0a0a13b6cb2c34e6bea6cc44c5cf4a96
- https://github.com/wso2/docs-apim/pull/2485/commits/091a414b2ea104063d6e44e88bf45785c6d9db3d

**Review by nuwand — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]
I feel that it will be good to fix the spacing between the text and the video before merging.

The following are the pending/new issues that I see, which are relatively trivial.

- Comment regarding the spacing/margins between capability icons and image. 

- In the key-manager-overview image the following two captions should start with capitals. (New comment)
  - access token > Access token
 - request token > Request token

- Image related comments.
    - Diagonal lines in the architecture pic
    - Getting the "Balance your APIs Demand" pics redrawn
